### PR TITLE
[CodeQualityStrict] Skip non native function on MoveVariableDeclarationNearReferenceRector

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba it is ready for review.

File: rules/CodeQualityStrict/Rector/Variable/MoveVariableDeclarationNearReferenceRector.php [hunk 1]
```diff
@@ -297,6 +298,15 @@ private function hasCall(Node $node): bool
...
             }
 
+            if (! function_exists($funcName)) {
+                return true;
+            }
+
+            $r = new ReflectionFunction($funcName);
+            if (! $r->isInternal()) {
+                return true;
+            }
```
Comment (TomasVotruba) [inline rules/CodeQualityStrict/Rector/Variable/MoveVariableDeclarationNearReferenceRector.php]: Here the `ReflectionProvider` should be used instead, so we have static reflection on board here :+1:

Comment (samsonasik) [reply rules/CodeQualityStrict/Rector/Variable/MoveVariableDeclarationNearReferenceRector.php]: implemented.

**Comment by samsonasik** [comment]
All checks have passed 🎉 @TomasVotruba I think it is ready.

**Comment by TomasVotruba** [comment]
Thank you :clap:

**Pull request merged**